core/server/data/migrations/versions/4.3/04-add-stripe-prices-table.js

```diff
@@ -0,0 +1,16 @@
+const addTable = require('../../utils').addTable;
+
+module.exports = addTable('stripe_prices', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    stripe_price_id: {type: 'string', maxlength: 255, nullable: false, unique: true},
+    stripe_product_id: {type: 'string', maxlength: 255, nullable: false, unique: false, references: 'stripe_products.stripe_product_id'},
+    active: {type: 'boolean', nullable: false},
+    livemode: {type: 'boolean', nullable: false},
```

Comment:
Is this necessary? No harm, just wondering the usecase :relaxed:

Reply:
😄 I was on the fence as well but decided to include because it _might_ come in as handy info when we pull in info for outside Ghost prices and needs a filtering on current connected mode or anything.